.github/workflows/release.yml

```diff
@@ -36,14 +36,14 @@ jobs:
           KEY_ALIAS: ${{ secrets.KEY_ALIAS }}
           KEY_PASSWORD: ${{ secrets.KEY_PASSWORD }}
           KEY_STORE_PASSWORD: ${{ secrets.KEY_STORE_PASSWORD }}
-        run: ./gradlew publishReleaseApk
+        run: ./gradlew publishReleaseBundle
 
       - name: Publish app to download.kiwix.org
         env:
-          UNIVERSAL_RELEASE_APK: app/build/outputs/apk/release/*universal*.apk
+          UNIVERSAL_RELEASE_AAB: app/build/outputs/apk/release/**.aab
```

Comment:
I think it is never going to work, as the path being provided here to locate application bundle is incorrect, as per the [_official_ article](https://medium.com/androiddevelopers/building-your-first-app-bundle-bbcd228bf631)

See:
![image](https://user-images.githubusercontent.com/56556919/127102598-f88d803c-f734-4315-ad40-56f4ba94e88b.png)

Reply:
Thanks!